### DISCUSSION TITLE
Exception Site List

### DISCUSSION THREAD

**Discussion by ke-od2024** [adoptium/adoptium-support#1202] [Q&A] [answered]
Is there a way to add an exception site list to an Eclipse Temurin JRE install similar to the Oracle Java Control Panel - Security - Exception Site List?

**Comment by karianna** [comment, marked as answer]
No, we don't support this use case.  You may wish to try OpenWebstart

**Reply by ke-od2024** [reply]
Thank you, and just so I'm clear there isn't a config file or any functionality to support this use case correct?

**Reply by karianna** [reply]
correct
